docs/tutorial/part-zero/index.md

```diff
@@ -1,325 +1,330 @@
 ---
-title: Set Up Your Development Environment
+title: Pystytä sinun kehitysympäristö
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Ennen kuin aloitat sinun ensimmäisen Gatsby-sivuston rakentamisen, sinun on perehdyttävä joihinkin ydin web-teknologioihin ja varmistettava, että olet asentanut kaikki vaadittavat ohjelmistotyökalut.
 
-## Familiarize yourself with the command line
+## Tutustu komentoriviin
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+Komentorivi on tekstipohjainen käyttöliittymä, jota käytetään komentojen suorittamiseen tietokoneellasi. Saatat myös nähdä, että sitä kutsutaan terminaaliksi. Käytämme tässä tutoriaalissa molempia keskenään vaihtokelpoisesti.
+Se on paljon kuin Finderin käyttö Macissa tai Resurssienhallinta Windowsissa. Finder ja Explorer ovat esimerkkejä graafisista käyttöliittymistä (GUI). Komentorivi on tehokas, tekstipohjainen tapa olla vuorovaikutuksessa tietokoneesi kanssa.
```

Comment:
Let's not add line breaks, because it will cause some troubles for us when there are incoming updates from the English repo.

Some suggestion for a bit more simple sentences and wording. Also the seconds `Explorer` word was not translated, I added that in the suggestion as well :)

```suggestion
Komentorivi on tekstipohjainen käyttöliittymä, jota käytetään komentojen suorittamiseen tietokoneellasi. Usein sitä voidaan kutsuta myös terminaaliksi. Käytämme tässä tutoriaalissa molempia vaihtokelpoisesti. Sen käyttö muistuttaa paljon Finderin käyttöä Macilla tai Resurssienhallintaa Windowsissa. Finder ja Resurssienhallinta ovat esimerkkejä graafisista käyttöliittymistä (GUI). Komentorivi on tehokas, tekstipohjainen tapa olla vuorovaikutuksessa tietokoneesi kanssa.
```

Reply:
Oops 😄. 

That's better 👍.